app/src/test/java/com/marekmacko/kotlinapp/ui/DailyForecastFragmentTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.marekmacko.kotlinapp.ui
+
+import android.support.v4.app.FragmentActivity
+import com.marekmacko.kotlinapp.BuildConfig
+import com.marekmacko.kotlinapp.MockData
+import kotlinx.android.synthetic.main.fragment_daily_forecast.*
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.android.controller.ActivityController
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(constants = BuildConfig::class)
+class DailyForecastFragmentTest {
+
+    private lateinit var fragment: DailyForecastFragment
+    private lateinit var activityController:ActivityController<FragmentActivity>
+
+    @Before
+    fun setUp() {
+        val forecast = MockData.getForecastList()[0]
+        fragment = DailyForecastFragment.newInstance(forecast)
+        activityController = Robolectric.buildActivity(FragmentActivity::class.java)
+        activityController.get()
+                .fragmentManager
+                .beginTransaction()
+                .add(fragment, null)
+                .commit()
+        activityController.create().start().resume().visible()
+    }
+
+    @Test
+    fun titleIsSetup() = assertEquals(MockData.DATE_STRING, fragment.activity.title)
+
+    @Test
+    fun forecastDataIsSetup() {
+        assertEquals(MockData.DESCRIPTION, fragment.descriptionView.text)
```

Comment:
I would recommend separate tests for these it is hard to follow what should be done with many asserts